lib/less/functions/number.js

```diff
@@ -71,6 +71,10 @@ functionRegistry.addMultiple({
         return new Dimension(Math.pow(x.value, y.value), x.unit);
     },
     percentage: function (n) {
+        if (typeof n !== "number") {
+            throw { type: "Argument", message: "argument must be a number" };
+        }
+        
```

Comment:
Thanks, but there are a number of problems.
First, it fails jscs - the line I'm commenting on has trailing whitespace. You can verify this with `grunt jscs`
Second, if you look below, we expect n to be of type Dimension, so n.value will be type number
Third, ideally it would be good to get a unit test for this - in the errors sub directory (run tests with `grunt test`)